src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
You're adding to the API but I don't think I saw a bump in the package version in cabal (I'd probably bump the minor version number).